puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/duffy.groovy

```diff
@@ -1,21 +1,41 @@
+import groovy.json.JsonSlurper
```

Comment:
Is this used anywhere?

Reply:
It's not I thought I had gotten rid of it, old artifact from when I was trying to parse the cico_data.json file.